test/api/movies-endpoint.js

```diff
@@ -0,0 +1,38 @@
+const ApiClient = require("../utilities/api-client");
+
+class MoviesEndpoint extends ApiClient {
+  constructor() {
+    super();
+    this.baseEndpoint = "/api/movies/";
+  }
+
+  async getPaginatedMovies(page = 0, size = 4) {
+    const params = { page, size };
+    return this.get(this.baseEndpoint, params);
+  }
+
+  async getMovieById(movieId) {
+    return this.get(`${this.baseEndpoint}${movieId}`);
```

Comment:
why did you use ` instead of ' or "?

Reply:
I used backticks ` instead of single ' or double " quotes because backticks allow me to directly insert variables using ${} syntax. This way, I can avoid concatenation when constructing strings that include dynamic values, like movieId in this case.